addons/sale/data/mail_template_data.xml

```diff
@@ -18,7 +18,8 @@
 % set access_action = object.with_context(force_website=True).get_access_action()
 % set doc_name = 'quotation' if object.state in ('draft', 'sent') else 'order confirmation'
 % set is_online = access_action and access_action['type'] == 'ir.actions.act_url'
-% set access_name = is_online and object.template_id and 'Accept and pay %s online' % doc_name or 'View %s' % doc_name
+% set pay_sign_name = 'pay' if object.require_payment == 1 else 'sign'
```

Comment:
Indeed as inheritance is not possible using jinja templates we have to perform some non-modular stuff. We could hide it in a method, but I don't think it would improve a lot the code. Do you ?